### ISSUE TITLE
Ability to remove preflight & default Normalize css configurations

### ISSUE THREAD

**Issue by transitorytammy** [embermap/ember-cli-tailwind#41]
It would be great if we could configure our ember-cli-tailwind setup to not include preflight by default and a way to exclude the normalize css styles because some of the default styles in etw.css are conflicting with my current styles.   Right now to deal with this, I am making these changes in a forked repo and including that in my app instead.

**Comment by samselikoff** [comment]
Yep, I need to change it so the styleguide is not built with a custom build of tailwind... it causes too many issues.

**Comment by Alonski** [comment]
I also need this 😄

**Comment by sschun123** [comment]
I am also in need of this

**Comment by leils** [comment]
Also needed here

**Comment by samselikoff** [comment]
Can you comment it out in `app/tailwind/modules.css`?

**Comment by novanda1** [comment]
`corePlugins: {
     preflight: false,
 }`

**Comment by raghav0622** [comment]
> `corePlugins: { preflight: false, }`

Doesnt works, reset styles are still injected.
